contract/r/gnoswap/gnft/svg_generator.gno

```diff
@@ -118,9 +118,33 @@ func paramsToImageURI(params string) string {
 }
 
 // isImageParams checks if the string is in the expected parameter format.
-// Expected format: "x1,y1,x2,y2,color1,color2" with exactly 5 commas.
+// Expected format: "x1,y1,x2,y2,color1,color2" where:
+//   - x1, y1, x2, y2 are non-negative numeric values (indices 0-3)
+//   - color1, color2 are hex color strings in #XXXXXX format (indices 4-5)
 func isImageParams(s string) bool {
-	return strings.Count(s, ",") == 5
+	if strings.Count(s, ",") != 5 {
+		return false
+	}
+
+	parts := strings.Split(s, ",")
+	if len(parts) != 6 {
+		return false
+	}
```

Comment:
```suggestion
	parts := strings.Split(s, ",")
	if len(parts) != 6 {
		return false
	}
```
If there are 5 commas, the split result will always have 6 parts, so the `Count` call appears to be redundant.